src/style.css

```diff
@@ -1 +1,52 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
Don't fix container size (if there is no such a requirement). It's usually better to let the content size dictate the dimensions of an element. If these dimensions are necessary for the design, then it's fine to keep them.